elections.md

```diff
@@ -1,55 +1,40 @@
-# Kubernetes Steering Committee Elections 
+# Kubernetes Steering Committee Elections
 
-This document outlines the process, for steering committee elections.  
+This document outlines the process, for steering committee elections.
 
-**NOTE:** This document is stale and requires reevaluation prior to next election cycle. 
-
-## Elections
-
-Special note: The bootstrap committee pledges to recuse itself from any direct
-election activities while they serve as continuity members. Members of the
-bootstrap committee will refrain from endorsing or otherwise advocating for any
-candidate (with the exception that the members of the bootstrap committee may
-vote in the elections, and may choose to run in the 2019 election).
-
-### Members of Standing
+### Eligibility for voting
 
-Initial standing in the Kubernetes community will be defined by the union of:
-* SIG leads
-* Approvers and reviewers in any Kubernetes owned repositories
-* Anyone with write access to a Kubernetes owned repository
+Eligibility to vote for steering committee members is defined by:
 
-The bootstrap committee *explicitly* believes that this heuristic will be
-inaccurate and not represent the entire community. An exception form will be
-provided for people who have contributed to the project but may not meet these
-criteria. Exception applications will be approved by the bootstrap committee
-will vote on this eligibility, with simple majority determining eligibility.
-The exception process will be used as data points for refining the criteria for
-the future.
+* People who have at least 50 [contributions to the Kubernetes project over the past year as defined by devstats]
+  as of 2018-08-15. Contributions include GitHub events like creating issues,
+  creating pr's, reviewing PR's, commenting on issues, etc. For full details
+  see [the SQL query used by devstats for developer stats]
+* People who have submitted the [voting exception form] and are accepted by
+  the election committee. Acceptance will be defined by a simple majority
+  vote, and the criteria used during this process will be used to help refine
+  further elections.
```

Comment:
Must the exception be filed every year? Do we need to publish the list of current exceptions?

Reply:
> Must the exception be filed every year? 

I would vote yes.  

> Do we need to publish the list of current exceptions?

I don't have strong opinions on that.  @castrojo - What did we do last time?

Reply:
@timothysc We had a public spreadsheet. This year, we would like to check the list in to either the steering repo or our events repo where the voters guide is held. This list would only be GH handles.

Reply:
I still think we should have some example valid exceptions, so people know when they should apply.  For example, is running a Kubernetes meetup a valid exception criterion?  Public speaking, and if so, how much?  What about code work on projects not in the kubernetes/ namespace, and if so which ones?

Reply:
Yes to filing every year.

The idea is that exceptions should be taken into account to further refine criteria next year.  If we're doing it right, exceptions (as a percentage of total eligible voters) should dwindle over time.

My concern with getting overly specific on examples is that they would then be taken as criteria. I would suggest leaving this to the election officers to decide/document rather than spelling out here.  I would also suggest leaving room for case-by-case interpretation, describing the spirit but not the letter.

Reply:
Well, I *am* saying that we should have exceptions-as-criteria.

Otherwise, what you're saying is that audacity is a criterion, as well as insider knowledge, which tends to result in a biased selection.

Reply:
It's not a matter of "having overly specific language"  It's a matter of "having some examples of any kind whatsoever".  Right now, our language is "file for an exception if you think you deserve an exception".

Reply:
This is the copy that is going on the top of the exception form. Criteria is for upstream activities (ie - running contributor workshops, doing contributor/community/governance talks specific to k8s, etc) 
"Note: If you were on the list in the announcement email, you do not need to complete this. 

We collected eligible voter information from DevStats, a CNCF Dashboard metrics site that collects events from all repositories in our GitHub organization(s). We understand that there may be exceptions here with other upstream activities that may not be listed as a contribution from GitHub events or for some other reason. Other upstream contributions outside of code and docs are encouraged to apply to vote. 

We will review and reach out to you by Monday, September 10th. We will be rolling updates through our voters guide throughout the election cycle."